src/index.js

```diff
@@ -113,6 +115,9 @@ export { GenericAd               } from './molecules/GenericAd';
 export { StickyAd                } from './molecules/StickyAd';
 export { LazyProgressiveImage    } from './molecules/LazyProgressiveImage';
 export { Source                  } from './molecules/Source';
+export { DetailsSection          } from './molecules/MainRecipe/DetailsSection';
+export { IngredientsSection      } from './molecules/MainRecipe/IngredientsSection';
+export { StepsSection            } from './molecules/MainRecipe/StepsSection';
```

Comment:
Does not return a `<section>`, and is limited to mapping `RecipeStep`s. How about we call it `RecipeSteps`?